src/status_im2/subs/shell.cljs

```diff
@@ -70,15 +83,15 @@
                            :profile-picture (when profile-picture
                                               (str profile-picture "&addRing=0"))}
      :customization-color (or (:customization-color contact) :primary)
-     :content             (get-card-content chat communities)
+     :content             (get-card-content chat communities false nil)
```

Comment:
It's up to you but it's just my personal preference, I think this function will look better if it's args are a in a map.

Reply:
addressed, thank you